input/resources/structuredefinition-encounter-description.xml

```diff
@@ -21,7 +21,7 @@
   <differential>
     <element id="Extension">
      <extension url="http://hl7.org/fhir/StructureDefinition/structuredefinition-standards-status">
-        <valueCode value="draft"/> 
+        <valueCode value="deprecated"/> 
       </extension> 
       <path value="Extension"/>
       <short value="Description, overview or summary of an encounter"/>
```

Comment:
Change the short and definition of this root element that this is deprecated and why. And example of the use of these fields is in: https://build.fhir.org/operationoutcome.html see OperationOutcome.issue.location.

Reply:
done